docs/documentation/js/release-info.js

```diff
@@ -0,0 +1,181 @@
+        mmmm: dF.i18n.monthNames[m + 12],
+        yy: String(y).slice(2),
+        yyyy: y,
+        h: H % 12 || 12,
+        hh: pad(H % 12 || 12),
+        H: H,
+        HH: pad(H),
+        M: M,
+        MM: pad(M),
+        s: s,
+        ss: pad(s),
+        l: pad(L, 3),
+        L: pad(L > 99 ? Math.round(L / 10) : L),
+        t: H < 12 ? "a" : "p",
+        tt: H < 12 ? "am" : "pm",
+        T: H < 12 ? "A" : "P",
+        TT: H < 12 ? "AM" : "PM",
+        Z: utc ? "UTC" : (String(date).match(timezone) || [""]).pop().replace(timezoneClip, ""),
+        o: (o > 0 ? "-" : "+") + pad(Math.floor(Math.abs(o) / 60) * 100 + Math.abs(o) % 60, 4),
+        S: ["th", "st", "nd", "rd"][d % 10 > 3 ? 0 : (d % 100 - d % 10 != 10) * d % 10]
+      };
+
+    return mask.replace(token, function ($0) {
+      return $0 in flags ? flags[$0] : $0.slice(1, $0.length - 1);
+    });
+  };
+}();
+
+// Some common format strings
+dateFormat.masks = {
+  "default": "ddd mmm dd yyyy HH:MM:ss",
+  shortDate: "m/d/yy",
+  mediumDate: "mmm d, yyyy",
+  longDate: "mmmm d, yyyy",
+  fullDate: "dddd, mmmm d, yyyy",
+  shortTime: "h:MM TT",
+  mediumTime: "h:MM:ss TT",
+  longTime: "h:MM:ss TT Z",
+  isoDate: "yyyy-mm-dd",
+  isoTime: "HH:MM:ss",
+  isoDateTime: "yyyy-mm-dd'T'HH:MM:ss",
+  isoUtcDateTime: "UTC:yyyy-mm-dd'T'HH:MM:ss'Z'"
+};
+
+// Internationalization strings
+dateFormat.i18n = {
+  dayNames: [
+    "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat",
+    "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"
+  ],
+  monthNames: [
+    "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec",
+    "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"
+  ]
+};
+
+// For convenience...
+Date.prototype.format = function (mask, utc) {
+  return dateFormat(this, mask, utc);
+};
+
+document.addEventListener("DOMContentLoaded", function () {
+  const url = 'https://flow.deckhouse.io/deployments';
+
+  fetch(url, {
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json;charset=utf-8'
+      },
+    })
+    .then(respose => respose.json())
+    .then(data => {
+      for (const item of data.deployments) {
+        const trBody = document.createElement('tr');
+        const channel = document.createElement('td');
+        const version = document.createElement('td');
+        const state = document.createElement('td');
+        const link = document.createElement('a');
+        const date = new Date(Date.parse(getValueCell(item, 'updated')))
+
+
+        channel.innerText = getValueCell(item, 'channel');
+        version.innerText = getValueCell(item, 'version');
+        link.href = getValueCell(item, 'log');
+        link.innerText = getValueCell(item, 'state');
+        link.style.paddingLeft = '5px';
+        state.innerText = ` at ${date.format("dddd, mmmm dd, yyyy")} at ${date.format("h:MM TT")}`;
```

Comment:
I see why we monkey-patch the [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) type. We could use Internationalization API to format dates. It would not require the boilerplate above. [`DateTimeFormat.format` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) is supported even in IE 😆 

```js
// formatDate(new Date())   ->   Wednesday, March 2, 2022 at 3:11 PM
function formatDate(date) {
		return new Intl.DateTimeFormat('en-US', {
			dateStyle: 'full',
			timeStyle: 'short'
		}).format(date);
	}
```